src/libraries/System.Transactions.Local/tests/NonMsdtcPromoterTests.cs

```diff
@@ -2164,6 +2165,7 @@ public void PSPENonMsdtcBlockingCloneCompletedAfterCommit(bool promote)
         [Theory]
         [InlineData(false)]
         [InlineData(true)]
+        [PlatformSpecific(~TestPlatforms.Browser)] // Cannot wait on monitors on this runtime.
```

Comment:
-> PlatformDetection.IsThreadingSupported

Reply:
Addressed in recent commits